packages/mobile/src/paymentRequest/PaymentRequestNotificationInner.tsx

```diff
@@ -17,7 +20,9 @@ export default function PaymentRequestNotificationInner(props: Props) {
       <Text style={[fontStyles.subSmall]}>
         {(requesterRecipient && requesterRecipient.displayName) || requesterE164Number} - {message}
       </Text>
-      <Text style={[fontStyles.subSmall, fontStyles.semiBold]}> {amount}</Text>
+      <Text style={[fontStyles.subSmall, fontStyles.semiBold]}>
+        {CURRENCIES[CURRENCY_ENUM.DOLLAR].symbol + getCentAwareMoneyDisplay(new BigNumber(amount))}
```

Comment:
Ditto here, no need to wrap in  BigNum before calling `getCentAwareMoneyDisplay`